tests/common/utilities.py

```diff
@@ -101,6 +105,8 @@ def run(self):
             threading.Thread.run(self)
         except Exception:
             self._e = sys.exc_info()
+            if self.error_handler:
```

Comment:
If a call to `set_error_handler` is not made, this Exception might throw a Python error where `error_handler` attribute is not present in `InterruptableThread`.

Reply:
Good catch.